Dockerfile

```diff
@@ -31,12 +31,12 @@ RUN cpanm install Git::Repository Data::Structure::Util
 ENV QUATTOR_TEST_TEMPLATE_LIBRARY_CORE /quattor/template-library-core-master
 
 # set workdir to where we'll run the tests
-COPY --chown=99 . /quattor_test
```

Comment:
why did you change this?

Reply:
That was not supposed to be committed. 

I need it to run the tests locally.